gravity.lisp

```diff
@@ -0,0 +1,70 @@
+;; Copyright (C) 2016 Joram Schrijver
```

Comment:
While I think its important to refactor into separate files, I think this is a bit too granular.  This functionality is really a user UI preference, and should get lumped in with similar preference type code.